### DISCUSSION TITLE
Ability to disable all notifiers

### DISCUSSION THREAD

**Discussion by CEikermann** [grafana/grafana#43946] [Alerting]
Hello,

Is there a way to just disable all notifiers for alerts in grafana?
Setting execute_alert to false in config leads that no alerting will popup in the UI anymore. We still want to see all alerts in the UI but just do not want to spread them out to all our notifiers

**Background:** We are restoring the grafana database from production towards our other environments to keep all dashboards, settings etc in sync across environments. The downside of the approach is that also notifiers are copied. This leads that notifiers are executed also for non-production env alerts.

**Comment by CEikermann** [comment]
Possible solution: I'm thinking of a config value (e.g. "execute_notifiers") which can be set to false which leads all alerts are still executed, just no notifier will be executed

**Comment by CEikermann** [comment]
https://grafana.com/docs/grafana/latest/administration/configuration/#max_attempts-1
```
max_attempts
Sets a maximum limit on attempts to sending alert notifications. Default value is 3.
```

Setting alerting.max_attempts to 0 leads into no alerting rule evaluation

**Comment by CEikermann** [comment]
Our current workaround of this is overwriting the specific hosts in the local host file from grafana, e.g. overwrite hooks.slack.com to 127.0.0.2 in order to have no slack webhooks.

**Reply by anavss** [reply]
Hi @CEikermann 👋🏼 one of our product designers is investigating usability improvements to the new Unified Alerting released in Grafana 8.3. I spotted your request and wondered if you would be interested in talking to her? You can [book a time with us here](https://ethn.io/75322). 🙂

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
